src/pack.rs

```diff
@@ -57,9 +64,72 @@ pub fn pack_instance(raw_module: Vec<u8>, mut ctor_module: elements::Module) ->
         function_internal_index
     };
 
+    let ret_function_id = {
+        let mut id = 0;
+        let mut found = false;
+        for entry in ctor_module.import_section().ok_or(Error::NoImportSection)?.entries().iter() {
+            if let External::Function(_) = *entry.external() {
+                if entry.field() == RET_SYMBOL { found = true; break; }
+                else { id += 1; }
+            }
+        }
+        if !found {
+            let mut mbuilder = builder::from_module(ctor_module);
+            let import_sig = mbuilder.push_signature(
+                builder::signature()
+                    .param().i32().param().i32()
+                    .build_sig()
+                );
+
+            mbuilder.push_import(
+                builder::import()
+                    .module("env")
+                    .field("ret")
+                    .external().func(import_sig)
+                    .build()
+                );
+
+            ctor_module = mbuilder.build();
+
+            let ret_func = ctor_module.import_count(ImportCountType::Function) as u32 - 1;
+
+            for section in ctor_module.sections_mut() {
+                match section {
```

Comment:
minor: we can deref `*section` and get rid of  `&mut` in `&mut elements::Section::Code` and so on